runtime/doc/eval.txt

```diff
@@ -3770,6 +3771,8 @@ feedkeys({string} [, {mode}])				*feedkeys()*
 		and "\..." notation |expr-quote|. For example,
 		feedkeys("\<CR>") simulates pressing of the <Enter> key. But
 		feedkeys('\<CR>') pushes 5 characters.
+		A special code that might be useful is <Ignore>, it exits the
+		wait for a character without doing anything.  *<Ignore>*
```

Comment:
Do we actually support this? Is there a test for it?

Reply:
It's for `K_IGNORE`. `git grep --files-with-matches '<Ignore>'` returns `runtime matches` so it's probably untested.

Reply:
I missed [8.1.1300](https://github.com/vim/vim/commit/2f10658b06bbdd8f25c4ff152266c808234cee0a)